doc/examples/README.rst

```diff
@@ -0,0 +1,138 @@
+.. -*- coding: utf-8 -*-
+
+========================
+ MDPOW Example: Benzene
+========================
+
+The ``examples`` directory contains input file for the solvation free
+energy calculation of benzene in solvent (water and octanol), using
+the OPLS-AA force field and the TIP4P water model.
+
+There are tow ways to use MDPOW:
```

Comment:
_tow_ should be _two_.

Reply:
thanks, 0124569845080456f40a02c718dacfe5ebd69c08